### PR TITLE
Add content about related projects

### LINKED ISSUE #2419: Add a page related to community projects based on netbox
### Change Type
[X] Addition
[ ] Correction
[ ] Deprecation
[ ] Cleanup (formatting, typos, etc.)

### Proposed Changes

Add a page related to community projects based on netbox

Example: 

https://github.com/sieben/netbox/blob/docs_community/docs/development/community.md

### PR DESCRIPTION
**Pull request by remyleone**
### Fixes:

Fix #2419

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -42,3 +42,19 @@ and run `upgrade.sh`.
...
+# Related projects
+
+## Supported SDK
+
+- [pynetbox](https://github.com/digitalocean/pynetbox) Python API client library for Netbox.
+
+## Community SDK
+
+- [netboxapi-client](https://github.com/bpetit/netboxapi-client) Python and CLI client for netbox
+- [python-netboxapi](https://github.com/Anthony25/python-netboxapi) Python client API for Netbox
```
Comment (cimnine) [inline README.md]: https://github.com/ninech/netbox-client-ruby/

**Comment by sieben** [comment]
The goal is to show the work of different contributors and create a community. If we restrict it to only one Python API client, people might not have complete information and access to an alternative if they miss a particular feature in the chosen one.

**Comment by jeremystretch** [comment]
There is no need to reference additional projects when there is already a supported client. Doing so only leads to confusion and complicates development efforts. It also increases our own maintenance burden as such projects pop up and die off with relative frequency.

**Comment by sieben** [comment]
Removed

**Pull request merged**